### PR TITLE
Cdpresolution/sync

### PR DESCRIPTION
**Pull request by ajaysikri**

### CODE REVIEW

File: src/connections/destinations/catalog/cdpresolution/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Setup within CDP Resolution:
+1.	From your [CDP Resolution Connectors](https://app.cdpresolution.com/administration/cdp-connections?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) page click on the Segment IO connector.
+2.	Paste your CDP Resolution API key in Segment to generate your Write key.
+3.	Paste your Write Key into CDP Resolution's connection configuration.
+4.	Click ‘Upload Key’.
+
+To verify that the destination has been set up correctly, check the Debugger section of your Segment Source. Assuming everything is as it should be, you should start seeing resolution data populate in the `identify` and `group` events – click on the specific event you’re interested in to see CDP Resolutions traits. These traits will now be available to other Segment destinations in your account. Notice that all CDP Resolution traits are prefixed with `cdpresolution_` to ensure they don’t conflict with existing traits.
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/cdpresolution/index.md:24]: Hi @ajaysikri , 

I don't believe that this is what the expected behavior of the Integration is. 

Please correct me if I'm wrong, but based on the code the following is what happens: 
When a event (such as a track() event) is triggered on the browser, CDP Resolution will set a cookie on the browser, and will send an API call to its servers containing the Segment anonymousId. 

Can you please clarify if this is the correct behavior?
Best regards,
Joe

**Review by joe-ayoub-segment — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/cdpresolution/index.md (hunk 1 above)
Comment (ajaysikri) [reply src/connections/destinations/catalog/cdpresolution/index.md:24]: Hi Joe,

You are correct from the description above.  However, we do have processing that takes place on the server side, in terms of what happens after we receive the anonymous Id.  And that is what we are stating in the above.  Our goal with this documentation was to provide a review of the complete offering that CDP Resolution is providing.

Please let me know if we need a call.

Thanks, Ajay

File: src/connections/destinations/catalog/cdpresolution/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+This destination is maintained by CDP Resolution. For any issues with the destination, [contact their Support team](mailto:support@cdpresolution.com).
+
+{% include content/ajs-upgrade.md %}
+
+## Getting started
+
+Setup within Segment:
+1.	From the Segment web app, click **Catalog**.
+2.	Search for “CDP Resolution” in the Catalog, select it, and choose which of your sources to connect the destination to.
+3.	In your Segment Settings UI, enter your CDP Resolution API key. You can find this in the CDP Connector Setting section of your [CDP Resolution Dashboard Connection Settings](https://app.cdpresolution.com/administration/cdp-connections/segment-io-f4241?utm_source=segmentio&utm_medium=docs&utm_campaign=partners).
```
Comment (stayseesong) [inline src/connections/destinations/catalog/cdpresolution/index.md:16]:
```suggestion
1.	Navigate to **Connections > Catalog** in the Segment app and select the **Destinations** tab of the catalog. 
2.	Search for *CDP Resolution* and select it.
3. Choose which of your sources to connect the destination to.
4.	In the Settings, enter your CDP Resolution API key. You can find this in the CDP Connector Setting section of your [CDP Resolution Dashboard Connection Settings](https://app.cdpresolution.com/administration/cdp-connections/segment-io-f4241?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"}.
5. Go to the CDP Resolution UI. 
5. Go to the [CDP Resolution Connectors](https://app.cdpresolution.com/administration/cdp-connections?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} page and select the Segment IO connector.
2.	Paste your CDP Resolution API key in Segment to generate your Write Key.
3.	Paste your Write Key into CDP Resolution's connection configuration.
4.	Click **Upload Key**.

```

File: src/connections/destinations/catalog/cdpresolution/index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Setup within CDP Resolution:
+1.	From your [CDP Resolution Connectors](https://app.cdpresolution.com/administration/cdp-connections?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) page click on the Segment IO connector.
+2.	Paste your CDP Resolution API key in Segment to generate your Write key.
+3.	Paste your Write Key into CDP Resolution's connection configuration.
+4.	Click ‘Upload Key’.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/cdpresolution/index.md:22]:
```suggestion
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/cdpresolution/index.md (hunk 1 above)
Comment (ajaysikri) [reply src/connections/destinations/catalog/cdpresolution/index.md:33]: Updated documentation based on discussion.

File: src/connections/destinations/catalog/cdpresolution/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+5. Go to the CDP Resolution UI. 
+5. Go to the [CDP Resolution Connectors](https://app.cdpresolution.com/administration/cdp-connections?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} page and select the Segment IO connector.
+2.	Paste your CDP Resolution API key in Segment to generate your Write Key.
+3.	Paste your Write Key into CDP Resolution's connection configuration.
+4.	Click **Upload Key**.
+
+Further documentation can be found on our [documentation site](https://docs.cdpresolution.com?utm_source=segmentio&utm_medium=docs&utm_campaign=partners).
+
+To verify that the destination has been set up correctly, check the Debugger section of your Segment Source. Assuming everything is as it should be, you should start seeing resolution data populate in the `identify` and `group` events – click on the specific event you’re interested in to see CDP Resolutions traits. These traits will now be available to other Segment destinations in your account. Notice that all CDP Resolution traits are prefixed with `cdpresolution_` to ensure they don’t conflict with existing traits.
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/cdpresolution/index.md:33]: Hi @ajaysikri , 
I think we should clarify this text a little more. 

For example: 

If you have configured your CDP Resolution Destination correctly, and if you've also configured CDP Resolution to send user profile data to a Segment Source, you should start to see user profile data show in the Segment Source debugger as identify() and group() calls. 

etc...

File: src/connections/destinations/catalog/cdpresolution/index.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+When you make requests to CDP Resolution, CDP Resolution sends events with its own data back to your Segment source server-side using Segment's Node JS Library. If you see unexpected traffic from curl in your Debugger, that traffic represents the events that CDP Resolution sends back.
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/cdpresolution/index.md:35]: Let's remove the second sentence as I think customers will find it confusing.

**Review by joe-ayoub-segment — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/cdpresolution/index.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+2.	Search for *CDP Resolution* and select it.
+3. Choose which of your sources to connect the destination to.
+4.	In the Settings, enter your CDP Resolution API key. You can find this in the CDP Connector Setting section of your [CDP Resolution Dashboard Connection Settings](https://app.cdpresolution.com/administration/cdp-connections/segment-io-f4241?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"}.
+5. Go to the CDP Resolution UI. 
+5. Go to the [CDP Resolution Connectors](https://app.cdpresolution.com/administration/cdp-connections?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} page and select the Segment IO connector.
+2.	Paste your CDP Resolution API key in Segment to generate your Write Key.
+3.	Paste your Write Key into CDP Resolution's connection configuration.
+4.	Click **Upload Key**.
+
+Further documentation can be found on our [documentation site](https://docs.cdpresolution.com?utm_source=segmentio&utm_medium=docs&utm_campaign=partners).
```
Comment (stayseesong) [inline src/connections/destinations/catalog/cdpresolution/index.md:31]:
```suggestion
Further documentation can be found on our [documentation site](https://docs.cdpresolution.com?utm_source=segmentio&utm_medium=docs&utm_campaign=partners).
```
```suggestion
Further documentation can be found on the [CDP documentation site](https://docs.cdpresolution.com?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"}.
```

File: src/connections/destinations/catalog/cdpresolution/index.md (hunk 5 above)
Comment (stayseesong) [inline src/connections/destinations/catalog/cdpresolution/index.md:35]:
```suggestion
When you make requests to CDP Resolution, CDP Resolution sends events with its own data back to your Segment source server-side using Segment's Node.js library. If you see unexpected traffic from curl in your Debugger, that traffic represents the events that CDP Resolution sends back.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/cdpresolution/index.md (hunk 4 above)
Comment (ajaysikri) [reply src/connections/destinations/catalog/cdpresolution/index.md]: Updated based on this feedback.

File: src/connections/destinations/catalog/cdpresolution/index.md (hunk 5 above)
Comment (ajaysikri) [reply src/connections/destinations/catalog/cdpresolution/index.md]: Updated based on this feedback.

Comment (ajaysikri) [reply src/connections/destinations/catalog/cdpresolution/index.md]: Removed this sentence based on feedback from Joe

**Review by stayseesong — APPROVED** [review]

**Pull request merged**